### PR TITLE
Camel

### LINKED ISSUE #74: consider changing dashes to camelCase in the openApi spec

### PR DESCRIPTION
**Pull request by tivvit**
Closes #74

### CODE REVIEW

File: example-bot/README.md [hunk 1]
```diff
@@ -59,14 +59,14 @@ configuration.api_key['SESSION_ID'] = 'YOUR_API_KEY'
 # configuration.api_key_prefix['SESSION_ID'] = 'Bearer'
 
 # create an instance of the API class
-api_instance = space_tycoon_client.GameApi(space_tycoon_client.ApiClient(configuration))
-body = NULL # dict(str, Command) | 
+api_instance = space_tycoon_client.DevApi(space_tycoon_client.ApiClient(configuration))
```
Comment (malytomas) [inline example-bot/README.md:62]: is there any practical motivation to separate the api?

Comment (tivvit) [reply example-bot/README.md:62]: there was some motivation to separate it. I do not know if it was practical.

Let me explain my motivation, and maybe you will have a good point not to do it.

I wanted to state very explicitly that this is a different part of the API which you are usually not going to use (and that it is not going to work). So it is not going to be present when you are reading the generated documentation and it is in a different section in the swagger docs visualization

**Review by evamayerova — APPROVED** [review]

File: example-bot/README.md (hunk 1 above)
Comment (evamayerova) [reply example-bot/README.md:62]: I think this was already done in another PR, this is just a newly generated client

Comment (malytomas) [reply example-bot/README.md:62]: > I think this was already done in another PR, this is just a newly generated client

There was a PR that has MERGED all api call into one.

Comment (malytomas) [reply example-bot/README.md:62]: > there was some motivation to separate it. I do not know if it was practical.
> 
> Let me explain my motivation, and maybe you will have a good point not to do it.
> 
> I wanted to state very explicitly that this is a different part of the API which you are usually not going to use (and that it is not going to work). So it is not going to be present when you are reading the generated documentation and it is in a different section in the swagger docs visualization

How about removing it completely from the swagger if it is not intended for use?

Comment (tivvit) [reply example-bot/README.md:62]: solved in person will remove it in the following PR

**Pull request merged**